include/seqan3/alignment/pairwise/policy/all.hpp

```diff
@@ -57,7 +57,7 @@
  * Function name     | Arguments                              | Return value                         |
  * ----------------- | -------------------------------------- | ------------------------------------ |
  * `compute_cell`    | `cell &&`, `cache &`, `score const &`  | `void`                               |
- * `make_cache`      | `gap_scheme const &`                   | alignment_algorithm_cache            |
+ * `make_cache`      | `gap_scheme const &`                   | alignment_algorithm_state            |
```

Comment:
```suggestion
 * `?make_cache?`      | `gap_scheme const &`                   | alignment_algorithm_state            |
```

Reply:
Well the entire documentation is not up to date anymore. I need to change this to actually document intrefaces. But I also have to move it to detail and thus need to make some directory changes there as well. I would then rework the documentation page for policies. But in a separate PR.